src/applications/disability-benefits/all-claims/tests/schema/maximal-test.json

```diff
@@ -206,9 +206,9 @@
       ],
       "view:militaryHistoryNote": {}
     },
-    "view:hasMilitaryRetiredPay": true,
+    "view:hasMilitaryRetiredPay": false,
```

Comment:
I think we need to leave these as true so that the test of the form data transform will transform the data for these pages.